src/core_landice/Registry.xml

```diff
@@ -567,6 +607,14 @@
             <var name="uReconstructX" packages="higherOrderVelocity"/>
 			<var name="uReconstructY" packages="higherOrderVelocity"/>
                         <var name="basalFrictionFlux"/>
+			<var name="damage"/>
+			<var name="ddamagedt"/>
```

Comment:
This section is variables that should be part of an initial condition to the model.  We definitely want to include `damage`, but I don't think any of the other variables you've added here should be input to the model (the damage code calculates them when it runs).  If that is the case, please delete these other variables from this section.

Reply:
Other variables than 'damage' are removed here